libr/debug/p/debug_native.c

```diff
@@ -1259,13 +1259,44 @@ static int r_debug_desc_native_open (const char *path) {
 	return 0;
 }
 
+static int r_debug_setup_ownership (int fd, RDebug *dbg) {
+	RDebugInfo *info = r_debug_info (dbg, "");
+
+	if (!info) {
+		eprintf ("Error while getting debug info.\n");
+		return -1;
+	}
+
+	if (fchown (fd, info->uid, info->gid) != 0) {
```

Comment:
no need to fail if ownership change fails. it's ok to save the file as r2 the user running r2. in fact, changing it can be problematic for the user in some environments